crypto/keyring/keyring.go

```
@@ -658,9 +641,10 @@ func SignWithLedger(k *Record, msg []byte, signMode signing.SignMode) (sig []byt
 		return nil, nil, errorsmod.Wrap(ErrInvalidSignMode, fmt.Sprintf("%v", signMode))
 	}
 
-	if !priv.PubKey().VerifySignature(msg, sig) {
-		return nil, nil, ErrLedgerInvalidSignature
-	}
+	// moved to injective-core as part of Ledger signing (PR #1557)
+	//if !priv.PubKey().VerifySignature(msg, sig) {
```

golangci-lint (1.64.8)

commentFormatting: put a space between `//` and comment text

(gocritic)